setup.py

```diff
@@ -29,7 +29,7 @@ def run_tests(self):
 # requirements
 install_requirements = ['guessit>=0.9.1', 'babelfish>=0.5.2', 'enzyme>=0.4.1', 'beautifulsoup4>=4.2.0',
                         'requests>=2.0', 'click>=4.0', 'dogpile.cache>=0.5.4', 'stevedore>=1.0.0',
-                        'chardet>=2.3.0', 'pysrt>=1.0.1', 'six>=1.9.0']
+                        'chardet>=2.3.0', 'pysrt>=1.0.1', 'six>=1.9.0', 'rarfile>=2.6']
```

Comment:
I'll check what's the required minimal version and use it. I know that the version 2.7 or 2.8 (I need to check) allows me to extract the file without using a temporary file, but I would rather stick with lower version dependency to increase compatibility.